server/modules/analysis/service/rChainService/index.js

```diff
@@ -42,7 +42,14 @@ export const persistResults = async (surveyId, cycle, stepUuid, filePath) => {
 
   await db.tx(async (tx) => {
     // Reset node results
-    await RChainManager.deleteNodeResults(surveyId, cycle, ProcessingStep.getProcessingChainUuid(step), tx)
+    await SurveyRdbMamager.deleteNodeResultsByChainUuid(
+      {
+        surveyId,
+        cycle,
+        chainUuid: ProcessingStep.getProcessingChainUuid(step),
+      },
+      tx
+    )
```

Comment:
declare chainUuid const above to fit this in one line

Reply:
ok